pom.xml

```diff
@@ -402,6 +403,66 @@
                     </exclusion>
                 </exclusions>
             </dependency>
+            <dependency>
+                <groupId>org.apache.maven</groupId>
+                <artifactId>maven-model</artifactId>
+                <version>${maven-core.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.apache.maven</groupId>
+                <artifactId>maven-core</artifactId>
+                <version>${maven-core.version}</version>
+                <exclusions>
+                    <exclusion>
+                        <groupId>javax.inject</groupId>
+                        <artifactId>javax.inject</artifactId>
+                    </exclusion>
+                    <exclusion>
+                        <groupId>org.slf4j</groupId>
+                        <artifactId>slf4j-api</artifactId>
+                    </exclusion>
+                </exclusions>
+            </dependency>
+            <dependency>
+                <groupId>org.apache.maven</groupId>
```

Comment:
@ppalaga hi is this dependency used somewhere and needed? It broke license generation, I though about updating licenses-generator but looking at it, embedder is used in it as some jenkins library which seems obsolete and I'm not feeling like digging in that. In short i removed it, tests are passing and my test builds with generation tests are working too.

Reply:
(Also I downgraded mvn to 3.6.3 because there was change in 3.8.x)